src/registry/content/Content.scss

```diff
@@ -0,0 +1,76 @@
+@use '@/scss/bem';
+@use '@utrecht/components/link';
+@use 'microscope-sass/lib/typography';
+
+$formio-content-variants: ('error', 'warning', 'success', 'info');
+
+@mixin formio-content-variant($variant) {
+  border-inline-start-color: var(
+    --of-formio-content-#{$variant}-border-inline-start-color,
+    var(--of-color-#{$variant})
+  );
+  border-inline-start-style: var(--of-formio-content-#{$variant}-border-inline-start-style, solid);
+  border-inline-start-width: var(--of-formio-content-#{$variant}-border-inline-start-width, 4px);
+  padding-inline-start: var(--of-formio-content-#{$variant}-padding-inline-start, 16px);
+}
+
+/**
+ * Extensions of the utrecht-link component with OF-specific styles.
+ */
+.utrecht-link {
+  @include bem.modifier('openforms') {
+    font-family: var(
+      --of-utrecht-link-font-family,
+      var(--of-typography-sans-serif-font-family, var(--utrecht-document-font-family))
+    );
+    margin: 0;
+  }
```

Comment:
I don't think the `font-family` rule is necessary - now would probably be a good time to use the standard NL DS design tokens.

Once we switch to the new renderer, I'm fairly certain we'll bump the SDK to a new major version (4.0) anyway so we can document the breaking changes.